packages/forklift-console-plugin/src/modules/Providers/views/list/components/ProvidersEmptyState.tsx

```diff
@@ -59,7 +59,7 @@ export const ProvidersEmptyState: React.FC<ProvidersEmptyStateProps> = ({
                   </TextListItem>
                   <TextListItem>
                     {t(
-                      'Map source datastores or storage domains or volume types and networks to target storage classes and networks.',
+                      'Map source datastores or storage domains or volume types or storage classes and networks to target storage classes and networks.',
```

Comment:
```suggestion
                      'Map source datastores, storage domains, volume types, storage classes, and networks to their respective target storage classes and networks.',
```

too many `or` , can we find a different way ?

Reply:
It's a legacy/not really used messages, so didn't invest on rephrasing, but will fix it.

Reply:
Done